crates/sim2h/Cargo.toml

```diff
@@ -24,13 +24,15 @@ lib3h_zombie_actor = "=0.0.25"
 holochain_tracing = "=0.0.4"
 holochain_core_types = { version = "=0.0.40-alpha1", path = "../core_types" }
 holochain_locksmith = { version = "=0.0.40-alpha1", path = "../locksmith" }
+in_stream = { version = "=0.0.40-alpha1", path = "../in_stream" }
 uuid = { version = "0.4", features = ["v4"] }
 log = "=0.4.8"
 env_logger = "=0.6.1"
 lazy_static = "=1.2.0"
 nanoid = "=0.2.0"
+num_cpus = "=1.11.1"
 url = "=2.1.0"
```

Comment:
Why do you still need the `url` crate explicitly? Shouldn't all `url::Url`s be `Url2` by now?

Reply:
: ( - alas, there's a lot of places that still expect the `url` crate type. e.g. Lib3hUri takes it, which is why i need to explicitly do a `Url::from(my_url2_instance).into()`